toolkits/code_sandbox/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "arcade_code_sandbox"
-version = "0.1.9"
+version = "1.0.0"
```

Comment:
I'm considering this a breaking change, because the required configuration has changed in a backwards-incompatible way.

cc @EricGustin for thoughts.

Reply:
sgtm. We also need to make the minimum arcade-ai version stricter for these toolkits

Reply:
Doh! fixed